seml/manage.py

```diff
@@ -340,3 +342,41 @@ def get_slurm_arrays_tasks():
             return {}
     except subprocess.CalledProcessError:
         return {}
+
+
+def get_nonempty_input(field_name, num_trials=3):
+    get_input = getpass if "password" in field_name else input
+    field = get_input(f"Please input the {field_name}: ")
+    trials = 0
+    while (field is None or len(field) == 0) and trials < num_trials:
+        logging.error(f'{field_name} was empty.')
+        field = get_input(f"Please input the {field_name}")
+        trials += 1
+    if field is None or len(field) == 0:
+        raise ValueError(f"Did not receive an input for {num_trials}. Aborting.")
```

Comment:
There should be an "times" in the sentence.

Also, in general I don't think that raising errors is helpful for the user, since this will print a stack trace. Imho we should almost always use `logging.error` instead.

Reply:
Ok, and how would you quit the execution then? Return `None` and then check in the calling function whether `None` was returned?

Reply:
Or just call `sys.exit()`.

Reply:
Ok, see 4b573e0.